site/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:14-alpine
+
+WORKDIR site
+
+COPY ./ /site
+COPY ../docs /site/src/docs
```

Comment:
I can't build it, unfortunately:

```
 > [4/5] COPY ../docs /site/src/docs:
------
failed to compute cache key: "/docs" not found: not found
```
Also, I don't know what would be the result given the fact that the run CMD sleeps.